src/renderer/views/Playlist/Playlist.js

```diff
@@ -387,16 +399,65 @@ export default defineComponent({
         this.isLoading = false
       }).catch((err) => {
         console.error(err)
-        if (process.env.SUPPORTS_LOCAL_API && this.backendPreference === 'invidious' && this.backendFallback) {
-          console.warn('Error getting data with Invidious, falling back to local backend')
-          this.getPlaylistLocal()
+        if (this.backendPreference === 'invidious' && this.backendFallback) {
+          if (process.env.SUPPORTS_LOCAL_API && this.fallbackPreference === 'local') {
+            console.warn('Error getting data with Invidious, falling back to local backend')
```

Comment:
Srry couldnt unsee this

```suggestion
            console.warn('Error getting data with Invidious, falling back to Local backend')
```